Atlas/backend/atlas/views.py

```diff
@@ -29,6 +29,10 @@ def create(self, request, *args, **kwargs):
 
         return super(cultural_heritage_item, self).create(request,*args,**kwargs)
 
+class comment(generics.ListCreateAPIView):
```

Comment:
There must be a queryset to retrieve all the comments on a spesific item

Reply:
Okey, I'll work on it